portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,7 +32,8 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private static final Gson GSON = new Gson();
-  private static DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static int numComment;
```

Comment:
It is not a good idea to use numComment as the global variable here, because is not thread safe.